elasticsearch/docker/config/performance-analyzer.properties

```diff
@@ -0,0 +1,25 @@
+## Paths
```

Comment:
I think we should not have a copy of performance-analyzer.properties here. In future, we might alter the file in our repo and forget it here. It should ideally reference the prop file in our repo! 

This file will be present in this location /usr/share/elasticsearch/plugins/opendistro-performance-analyzer/pa_config. 

Any reason on why we are adding it here? Any thoughts about this @ktkrg?

Reply:
This is a customization of the config file from the PA repo. Specifically, we *need* to bind to 0.0.0.0, which the default performance-analyzer.properties does not do.